src/Tintin.hs

```diff
@@ -25,9 +26,9 @@ runApp outputDirectory = do
   filenames <- getDocumentationFilenames docDir
 
   Parse.docs docDir filenames
-   |>> Render.perform
-   |>> ConfigurationLoading.loadInfo
-   |>> Render.writeOutput outputDirectory
+   >>= Render.perform
+   >>= CL.loadInfo
+   >>= Render.writeOutput outputDirectory
```

Comment:
In fact, I simply rewrote this in terms of bind, but this could be further polished into a do-block, which is completely standard in any Haskell introductory material. On the other hand, if you want to gear the repo towards incoming F# or Scala programmers, `(|>)` would be straightforward to them.

Reply:
Definitely stuff is confusing, and I'm gearing towards of getting into some middle ground, by removing the F# style operators and then just using `&` from `Data.Function` (which is imported in Universum by default btw) and bind.